### PR TITLE
Update secrets mgmt

### PR DESCRIPTION
**Pull request by Sgtpluck**
Addresses #138. 

I also tried to clean up the page a bit to make it more readable and add classification tabs (which I could have sworn was an issue but I can't find it now.)

### CODE REVIEW

File: _pages/workflow.md
Comment (ryanhofdotgov) [inline _pages/workflow.md:42]:
```suggestion
accidentally publishing it. (This repo assumes macOS with Homebrew installed.)
```

File: _pages/workflow.md
Comment (ryanhofdotgov) [inline _pages/workflow.md:44]:
```suggestion
{%include components/tag-requirement.html %} Enable 
```

**Review by ryanhofdotgov — APPROVED** [review]
I think that the use of tags in this manner might reduce readability, but don't feel strongly enough to hold up the PR. Maybe get another pair of eyes to weigh in?

**Comment by Sgtpluck** [comment]
@ryanhofdotgov i agree it's a bit distracting, so i reorganized it to cluster in terms of standards vs suggestions, i think that reads better! thank you.

**Comment by Sgtpluck** [comment]
i'm curious if you think those tags need some attention-- i don't find them very intuitive? for instance, i would say "default" instead of "standard;" and "suggestion" seems to mean that "this has worked in the past but we don't necessarily recommend it" which is not what that word means to me.

**Comment by ryanhofdotgov** [comment]
@Sgtpluck Yeah, I think it is worth re-iterating on. I haven't really found used them beyond a bullet list of products or for a whole section. Here's a couple of the PRs where the original discussion took place:
* https://github.com/18F/development-guide/pull/83
* https://github.com/18F/development-guide/pull/108
* https://github.com/18F/development-guide/pull/218

**Pull request merged**
